doc/index.rst

```diff
@@ -2,6 +2,219 @@
+   INFO:ceph-daemon:Waiting for mon to start...
+   INFO:ceph-daemon:Assimilating anything we can from ceph.conf...
+   INFO:ceph-daemon:Generating new minimal ceph.conf...
+   INFO:ceph-daemon:Restarting the monitor...
+   INFO:ceph-daemon:Creating mgr...
+   INFO:ceph-daemon:Creating crash agent...
+   Created symlink /etc/systemd/system/ceph-335b6dac-064c-11ea-8243-48f17fe53909.target.wants/
+   ceph-335b6dac-064c-11ea-8243-48f17fe53909-crash.service → /etc/systemd/system/ceph-335b6dac-
+   064c-11ea-8243-48f17fe53909-crash.service.
+   INFO:ceph-daemon:Wrote keyring to ceph.keyring
+   INFO:ceph-daemon:Wrote config to ceph.conf
+   INFO:ceph-daemon:Waiting for mgr to start...
+   INFO:ceph-daemon:mgr is still not available yet, waiting...
+   INFO:ceph-daemon:mgr is still not available yet, waiting...
+   INFO:ceph-daemon:Generating ssh key...
+   INFO:ceph-daemon:Wrote public SSH key to to ceph.pub
+   INFO:ceph-daemon:Adding key to root@localhost's authorized_keys...
+   INFO:ceph-daemon:Enabling ssh module...
+   INFO:ceph-daemon:Setting orchestrator backend to ssh...
+   INFO:ceph-daemon:Adding host 192-168-1-101.tpgi.com.au...
+   INFO:ceph-daemon:Enabling the dashboard module...
+   INFO:ceph-daemon:Waiting for the module to be available...
+   INFO:ceph-daemon:Generating a dashboard self-signed certificate...
+   INFO:ceph-daemon:Creating initial admin user...
+   INFO:ceph-daemon:Fetching dashboard port number...
+   INFO:ceph-daemon:Ceph Dashboard is now available at:
+   URL: https://192-168-1-101.tpgi.com.au:8443/
+   User: admin
+   Password: oflamlrtna
+   INFO:ceph-daemon:You can access the Ceph CLI with:
+   sudo ./ceph-daemon shell -c ceph.conf -k ceph.keyring
+   INFO:ceph-daemon:Bootstrap complete.
+   END QUOTED TEXT
+
+
+Second Node
+-----------
+
+#. Install a recent, supported Linux distribution on a second computer.
+#. Install podman, docker, and python::
+
+    $ sudo dnf install podman docker python
+#. Turn on ssh on node 2:::
+
+    $ sudo systemctl start sshd
+    $ sudo systemctl enable sshd
+#. Create a file on node 2 that will hold the ceph public key::
+
+    $ sudo mkdir -p /root/.ssh
+    $ sudo touch /root/.ssh/authorized_keys
+#. Copy the public key from node 1 to node 2::
+
+    [node 1] $ sudo cat ceph.pub | ssh root@192.168.1.102 tee -a /root/.ssh/authorized_keys
+
+Third Node
+----------
+#. Install a recent, supported Linux distribution on a third computer.
+
+#. Install podman, docker, and python::
+
+    $ sudo dnf install podman docker python
+#. Turn on ssh on node 3::
+
+    $ sudo systemctl start sshd
+    $ sudo systemctl enable sshd
+#. Create a file on node 3 that will hold the ceph public key::
+
+    $ sudo mkdir -p /root/.ssh
+    $ sudo touch /root/.ssh/authorized_keys
+
+#. Copy the public key from node 1 to node 3::
+
+    [node 1] $ sudo cat ceph.pub | ssh root@192.168.1.103 tee -a /root/.ssh/authorized_keys
+#. On node 1, issue the command that adds node 3 to the cluster::
```

Comment:
this step is needed on the 2nd node too, and should be run in a './cephadm shell' so it has access to the ceph command - same with later ceph commands

Reply:
@jdurgin I believe that I've gotten this change in correctly now, but my eye has been up against the stone so closely that I've gone facet-blind. Let me know if it's right.

Reply:
I meant the command below, 'ceph orchestrator host add'. Copying the ssh key doesn't use any ceph commands, so it can be run on the host directly. To be clear, any 'ceph ...' command should be run in the cephadm shell, so this becomes:

```[node 1] $ sudo ./cephadm shell -c ceph.conf -k ceph.keyring
[ceph: root@node 1] $ ceph orchestrator host add 192.168.1.103
```

Reply:
You've got a description of entering the ceph shell at the end, perhaps that could move earlier and you could have some shorthand for it like [ceph shell] or something, since there are several 'ceph orchestrator...' commands in the process

Reply:
We'll need to hash this one out line-by-line in person.